api/src/pcapi/utils/pdf_creation.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+from weasyprint import HTML
+
+
+PDF_AUTHOR = "Pass Culture"
+
+
+@dataclass
+class PdfMetadata:
+    title: str = ""
+    description: str = ""
+    author: str = PDF_AUTHOR
+
+
+def generate_pdf_from_html(html_content: str, metadata: PdfMetadata = PdfMetadata()) -> bytes:
```

Comment:
L'instance est mutable, mais je suppose qu'on ne risque rien à la définir comme valeur par défaut. Pour l'instant, en tout cas.

Variante possible : 

```suggestion
def generate_pdf_from_html(html_content: str, metadata: PdfMetadata = None) -> bytes:
  metadata = metadata or PdfMetadata()
```

Reply:
Bien vu, soyons prudents.